tests/test_contamination_tech_reps.nf.test.snap

```diff
@@ -61,36 +61,36 @@
             true,
             true,
             true,
-            "Clone1_N1_trimmed_mature.sorted.idxstats:md5,761f06609a8c9b2ac05fe1505bf7f490",
```

Comment:
How and why are these files different now?

Reply:
The obtained reads from the contaminant filter are assigned to the channel with reads for `MIRNA_QUANT`:
`ch_reads_for_mirna = CONTAMINANT_FILTER.out.filtered_reads`. So, any change in the contaminant filter will be reflected here.

Reply:
Files are non-deterministic, disregard the comment